src/menu/index.tsx

```diff
@@ -14,7 +14,7 @@ import * as css from '../theme/default/menu.m.css';
 import ListBoxItem from './ListBoxItem';
 import MenuItem from './MenuItem';
 
-export type MenuOption = { value: string; label?: string; disabled?: boolean };
+export type MenuOption = { value: string; label?: string; disabled?: boolean; divider?: boolean };
```

Comment:
I think we should add details of these options to the `options` property description or to the README so that the display on the docs.

Reply:
I added a description of these properties to the `options` comment